examples/react/json-based-microapp-renderer-with-custom-components/src/components/Microapp.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2022 Workgrid Software
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react'
+import { useQuery, gql } from '@apollo/client'
+import Loader from './Loader'
+import Card from './Card'
+
+// GraphQL Query for microapp content
+const GET_MICROAPP_QUERY = gql`
+  query GetMicroapp($spaceId: ID!, $entrypoint: String!, $data: JSONObject) {
+    me {
+      space(spaceId: $spaceId) {
+        appView(entrypoint: $entrypoint, data: $data) {
+          view
+        }
+      }
+    }
+  }
+`
+
+const Microapp = ({ node, key }) => {
```

Comment:
`key` is a special react prop that does not get passed to the child component.

> Keys serve as a hint to React but they don’t get passed to your components.
https://reactjs.org/docs/lists-and-keys.html

Reply:
Ah, thank you - Removed! 👍